crates/astria-composer/src/strategy.rs

```diff
@@ -0,0 +1,22 @@
+use crate::ds::RollupTxExt;
+use async_trait::async_trait;
+
+/// The Strategy trait defines arbitrary MEV strategies executed on a list of received
+/// rollup transactions
+#[async_trait]
+pub(crate) trait Strategy{
```

Comment:
I dont see the need for this right now - this should be a separate, self-contained pr for abstracting over the strategy

Reply:
I can split it up into another PR, but I'm not sure if that's really necessary? It will just delete 5 lines of code max